lib/node_modules/@stdlib/assert/deep-has-own-property/lib/deep_has_own_property.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
```

Comment:
To simplify filenames a bit, can you rename this file to `main.js`? (will need to update any `require` statements)